platform/ui/src/components/Tooltip/Tooltip.tsx

```diff
@@ -69,20 +74,20 @@ const Tooltip = ({
       >
         <div
           className={classnames(
-            'relative tooltip-box bg-primary-dark border border-secondary-main text-white text-base rounded inset-x-auto top-full w-max-content',
+            'relative tooltip-box bg-primary-dark border border-secondary-light text-white text-base rounded inset-x-auto top-full w-max-content',
             {
               'py-1 px-4': !tight,
             }
           )}
         >
           {typeof content === 'string' ? t(content) : content}
           <svg
-            className="absolute h-4 text-primary-dark stroke-secondary-main"
+            className="absolute h-4 text-primary-dark stroke-secondary-light"
             style={arrowPositionStyle[position]}
             xmlns="http://www.w3.org/2000/svg"
             viewBox="0 0 24 24"
           >
-            <path fill="currentColor" d="M24 22h-24l12-20z" />
+            <path fill="currentColor" d="M24 22l-12-20l-12 20" />
```

Comment:
The arrow that points out of the tooltip was a triangle before. With some css trickery one line would be hidden so that it looked like the arrow was part of the tooltip border. But if we are hiding one line anyways, why not just have two? And that is what this change does. I was sort of "forced" to make this change because depending on how things are positioned there maybe rounding in the positioning which makes it more difficult to "cover up" the third line.